dist/bin/reacli.js

```diff
@@ -205,13 +241,17 @@ var reactCli = function reactCli(args) {
 	var options = {};
 	var useFlow = /\x2D\x2Dflow|\x2Df/.test(args);
 	var useScss = /\x2D\x2Dscss/.test(args);
+	var useRedux = /\x2D\x2Dredux|\x2Dr/.test(args);
```

Comment:
there is a very similar piece of code above: ` const useRedux = (/--redux|-r/u).test(args)`.
maybe it should be extracted to a separate function?